charts/komodor-agent/values.yaml

```diff
@@ -0,0 +1,328 @@
+# @ignored
```

Comment:
what is @ignored?

Reply:
This is related to the helm-docs tool that we are using to generate the ReadMe and enforce values documentation.
The [@ignore](https://github.com/norwoodj/helm-docs#ignoring-values) is to tell the helm-docs not to add this part to the readme.